src/pages/venues/[id]/map/map.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "breakpoints";
+@import "variables";
+
+.content {
+  --background-color: #{$color-engel-medium-light};
+  grid-column: 1 / -1 !important;
+}
+
+.title {
+  font-size: 24px;
```

Comment:
Could use a design token from HDS.

Applies more generally to this file as well, `1rem` `$spacing-m` and `color: black` could be `$color-black`.